std/assembly/set.ts

```diff
@@ -118,14 +118,63 @@ export class Set<T> {
     }
   }
 
-  @operator("[]=")
-  private __set(key: T, value: bool): void {
-    if (value) this.add(key);
-    else this.delete(key);
```

Comment:
Wait! It seems you accidentality removed `__set` method!

Reply:
I recovered it. I'm sorry to remove it accidentally.